### DISCUSSION TITLE
Chromebook setup with Termux: getting error message

### DISCUSSION THREAD

**Discussion by renChenTheProgrammer** [community/community#35286] [Other Feature Feedback, Questions, & Ideas]
Hi all, I'm trying to set up Termux so I can use GitHub from my Chromebook. Here's what's happening:

1. I type "apt install git" like the instructions say
2. An error says "Unable to locate package git"

I've searched online and tried a couple fixes, but nothing has allowed me to install git on Termux. I also didn't see any discussions on here about it. Any ideas or suggestions? Should I just find a different download that works similarly to Termux?

Thank you!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @misslaurenchen! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by daeho-ro** [comment]
Maybe you need `pgk update` or `apt update`?

**Comment by jfullstackdev** [comment]
as far as I can remember, there was also one user asking about the setup, seems Chromebook is not yet standard, I did not know what happened to that user because it was error followed by errors, maybe he/she followed my advise 

my only suggestion was that, buy a cheap laptop with Windows installed, so that it will not be difficult to install everything, how much more if that will be a true development, like when installing a framework, say, Laravel
